docker-compose.yml

```diff
@@ -16,13 +16,14 @@ services:
       S2S_KEY: S2S_KEY
       CCD_DOCUMENT_API_IDAM_KEY:
       USER_PROFILE_HOST:
+      Document_STORE_URL: http://ccd-api-gateway:3453/documents
       AZURE_APPLICATIONINSIGHTS_INSTRUMENTATIONKEY:
       IDAM_USER_URL: http://idam-api:5000
       IDAM_S2S_URL: http://service-auth-provider-api:8080
-      #JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0 -XX:MinRAMPercentage=30.0 -XX:+UseConcMarkSweepGC -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005
+      JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0 -XX:MinRAMPercentage=30.0 -XX:+UseConcMarkSweepGC -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005
```

Comment:
We should keep the Java tool options disabled and let the container decide the memory profile for the application

Reply:
It won't impact the container, hence keeping as is.

Reply:
It's doesn't affect on any functionality.